app/wizard.py

```diff
@@ -45,7 +45,9 @@ def __init__(self, mypkcs, myacme):
 if __name__ == "__main__":
     app = QApplication(sys.argv)
     pkcs = pkcs()
-    acme = ACME("https://acme.proeftuin.uzi-online.rdobeheer.nl/")
+
+    # This expects the new ACME server to run on this port. Later on, make this configurable
+    acme = ACME("http://localhost:8080/")
```

Comment:
This will already picked up by environment file part of the [documentation ticket](https://github.com/minvws/nl-rdo-uzi-coordination-private/issues/300).